flink-cdc-e2e-tests/flink-cdc-pipeline-e2e-tests/pom.xml

```diff
@@ -41,6 +41,8 @@ limitations under the License.
         <flink.release.name>flink-${flink.version}-bin-scala_${scala.binary.version}.tgz</flink.release.name>
         <flink.release.mirror>https://dlcdn.apache.org/flink/flink-${flink.version}</flink.release.mirror>
         <maven.plugin.download.version>1.6.8</maven.plugin.download.version>
+        <iceberg.version>1.6.1</iceberg.version>
+        <hive.version>2.3.9</hive.version>
```

Comment:
minor: we can use version property to avoid dIff between test and production codebase